management/vmutils/network.go

```diff
@@ -42,15 +42,15 @@ func ConfigureWithExternalPort(role *Role, name string, localport, externalport
 	role.ConfigurationSets = updateOrAddConfig(role.ConfigurationSets, ConfigurationSetTypeNetwork,
 		func(config *ConfigurationSet) {
 			if config.InputEndpoints == nil {
-				config.InputEndpoints = &[]InputEndpoint{}
+				config.InputEndpoints = []InputEndpoint{}
```

Comment:
is this needed? I think it will work exactly the same without initialization http://play.golang.org/p/upFJkl5T5S